docs/docs/contributing/contributing.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Contributing guidelines
+permalink: /contributing/contributing
+---
+
+# Contributing
+
+Thank you for considering contributing to Gosemble! We welcome contributions from everyone. To maintain a high standard of quality, please follow these guidelines when contributing to the project.
+
+## Signing Your Commits
+
+To contribute to Gosemble, you must have a verified GPG key added to your GitHub account. This ensures the integrity and authenticity of your contributions. For more information on adding a GPG key to your GitHub account, refer to [GitHub's official documentation](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/adding-a-new-gpg-key-to-your-github-account).
+
+## Commit Message Guidelines
+
+We follow the [Conventional Commits specification](https://www.conventionalcommits.org/) for our commit messages. Each commit message should have a clear and structured format that conveys the type, scope, and description of the changes made. This helps in understanding the purpose of each commit and automating tasks like versioning and changelog generation.
+
+## Merge Process
+
+When submitting a new pull request (PR), please ensure the following:
+
+- **Target the `develop` branch**: All new PRs must target the `develop` branch.
+- **Use our PR template**: Utilize our [PR template](/.github/PULL_REQUEST_TEMPLATE.md) to ensure all necessary information is included.
```

Comment:
the link does not work when accessed from [limechain.github.io/gosemble/](https://limechain.github.io/gosemble/), you need to provide the full path (check `api.md` as an example)